go.mod

```diff
@@ -4,6 +4,7 @@ go 1.22.0
 
 require (
 	filippo.io/mkcert v1.4.4
+	fybrik.io/crdoc v0.6.3
```

Comment:
We used this one for cert-manager projects https://github.com/cert-manager/website/blob/master/scripts/bin/crdoc#L42-L53
Another common alternative is https://github.com/ahmetb/gen-crd-api-reference-docs, but that is in maintenance mode and has some issues around displaying links